src/util/initialState.js

```diff
@@ -60,6 +60,7 @@ export const initialState = () => {
     schemaVersion: SCHEMA_LATEST,
     showHiddenThoughts: false,
     showSidebar: false,
+    splitPosition: localStorage.getItem('splitPosition') ? JSON.parse(localStorage.getItem('splitPosition')) : 0,
```

Comment:
This is a bit cleaner and does not involve two calls to `getItem`:

```js
splitPosition: JSON.parse(localStorage.getItem('splitPosition') || '0')
```

Reply:
Good catch. Will update it